test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'by ID' do
```

Comment:
`it 'requires ID'` or `'accepts ID'` would be better. Remember that this should read like a sentence. i.e. "SyncPlan::InfoCommand accepts ID" rather than "SyncPlan::InfoCommand by ID"